tests/test_branch_stats.py

```diff
@@ -343,6 +486,16 @@ def test_windowization(self):
         ts = msprime.simulate(10, random_seed=self.random_seed, recombination_rate=100)
         self.check_windowization(ts)
 
+    def test_derived_functions(self):
+        '''
```

Comment:
Can you change this to a '#' comment please? Docstrings for test functions are displayed on the console when you run ``nosetests -v``, which make it difficult to see what test is actually being run. This is the only time when docstrings are bad though!